src/auxiliary/special_elixirs.jl

```diff
@@ -59,7 +58,8 @@ This function assumes that the spatial resolution is set via the keywords
 integers, one per spatial dimension).
 """
 function convergence_test(mod::Module, elixir::AbstractString, iterations; kwargs...)
-  @assert(iterations > 1, "Number of iterations must be bigger than 1 for a convergence analysis")
+  @assert(iterations>1,
+          "Number of iterations must be bigger than 1 for a convergence analysis")
```

Comment:
This is even more weird that the other `@assert` statements since there is definitely no ambiguity when adding braces around the list of arguments.

Reply:
Agreed. Do you know if this only happens in macros and for comparison operators, or for any binary operator?

Reply:
I guess it's only for macros